src/NBitcoin/Network.cs

```diff
@@ -10,1429 +12,1079 @@
-				throw new ArgumentNullException("dest");
-			return new BitcoinPubKeyAddress(dest, this);
-		}
-
-		private BitcoinAddress CreateBitcoinScriptAddress(ScriptId scriptId)
-		{
-			return new BitcoinScriptAddress(scriptId, this);
-		}
-
-		public Message ParseMessage(byte[] bytes, ProtocolVersion version = ProtocolVersion.PROTOCOL_VERSION)
-		{
-			BitcoinStream bstream = new BitcoinStream(bytes);
-			Message message = new Message();
-			using(bstream.ProtocolVersionScope(version))
-			{
-				bstream.ReadWrite(ref message);
-			}
-			if(message.Magic != magic)
-				throw new FormatException("Unexpected magic field in the message");
-			return message;
-		}
-
-#if !NOSOCKET
-		public IEnumerable<NetworkAddress> SeedNodes
-		{
-			get
-			{
-				return this.vFixedSeeds;
-			}
-		}
-		public IEnumerable<DNSSeedData> DNSSeeds
-		{
-			get
-			{
-				return this.vSeeds;
-			}
-		}
-#endif
-		public byte[] _MagicBytes;
-		public byte[] MagicBytes
-		{
-			get
-			{
-				if(_MagicBytes == null)
-				{
-					var bytes = new byte[]
-					{
-						(byte)Magic,
-						(byte)(Magic >> 8),
-						(byte)(Magic >> 16),
-						(byte)(Magic >> 24)
-					};
-					_MagicBytes = bytes;
-				}
-				return _MagicBytes;
-			}
-		}
-		public uint Magic
-		{
-			get
-			{
-				return magic;
-			}
-		}
-
-		public Money GetReward(int nHeight)
-		{
-			long nSubsidy = new Money(50 * Money.COIN);
-			int halvings = nHeight / consensus.SubsidyHalvingInterval;
-
-			// Force block reward to zero when right shift is undefined.
-			if(halvings >= 64)
-				return Money.Zero;
-
-			// Subsidy is cut in half every 210,000 blocks which will occur approximately every 4 years.
-			nSubsidy >>= halvings;
-
-			return new Money(nSubsidy);
-		}
-
-		public bool ReadMagic(Stream stream, CancellationToken cancellation, bool throwIfEOF = false)
-		{
-			byte[] bytes = new byte[1];
-			for(int i = 0; i < MagicBytes.Length; i++)
-			{
-				i = Math.Max(0, i);
-				cancellation.ThrowIfCancellationRequested();
-
-				var read = stream.ReadEx(bytes, 0, bytes.Length, cancellation);
-				if(read == 0)
-					if(throwIfEOF)
-						throw new EndOfStreamException("No more bytes to read");
-					else
-						return false;
-				if(read != 1)
-					i--;
-				else if(_MagicBytes[i] != bytes[0])
-					i = _MagicBytes[0] == bytes[0] ? 0 : -1;
-			}
-			return true;
-		}
-	}
+    public class DNSSeedData
+    {
+        private IPAddress[] addresses;
+
+        public string Name { get; }
+
+        public string Host { get; }
+
+        public DNSSeedData(string name, string host)
+        {
+            this.Name = name;
+            this.Host = host;
+        }
+
+        public IPAddress[] GetAddressNodes()
+        {
+            if (this.addresses != null)
+            {
+                return this.addresses;
+            }
+
+            try
+            {
+                this.addresses = Dns.GetHostAddressesAsync(this.Host).Result;
+            }
+            catch(AggregateException ex)
```

Comment:
when fixing code style, i always do delete last `}` in the file and then write it there again. MSVS will format the whole code of the file as per its rules. It should change this line to `catch (AggregateException ex)` - i.e. insert the space